<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>637</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>1500</width>
        <height>1400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 241, 227);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_5" native="true">
         <property name="maximumSize">
          <size>
           <width>900</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 85, 255);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1" alignment="Qt::AlignTop">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Times New Roman&quot;;
background-color:rgb(27, 154, 170);
color:rgb(221, 219, 203);
border-radius:4px solid beige;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Cart Items&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QTableWidget" name="DataTable">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(245, 241, 227);

</string>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="cornerButtonEnabled">
             <bool>true</bool>
            </property>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Price</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>city</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Model</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>cc</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Mode</string>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="0" alignment="Qt::AlignLeft">
           <widget class="QPushButton" name="RemoveFromCartbtn">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
background-color:rgb(27, 154, 170);
color:rgb(221, 219, 203);
}

QPushButton:hover{
	
	background-color: rgb(245, 241, 227);
	
	color: rgb(0, 0, 0);
	border: 2px solid grey;
}

</string>
            </property>
            <property name="text">
             <string>Remove From Cart</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Times New Roman&quot;;
background-color:rgb(27, 154, 170);
color:rgb(221, 219, 203);
border-radius:4px solid beige;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Total Price&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" alignment="Qt::AlignRight">
           <widget class="QLineEdit" name="lineEdit">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Times New Roman&quot;;
color:rgb(27, 154, 170);
background-color:rgb(221, 219, 203);
border:2px solid beige;
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>1400</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>150</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../../final projext/240_F_103293921_HnSUxsEatUrgvdOp57IuriUeMrpRlMrt-removebg-preview.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(27, 154, 170);
color:rgb(221, 219, 203);
font-family:times new roman;


</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;ORDER SUMMARY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
